builtin/mainmenu/tab_online.lua

```diff
@@ -65,13 +65,15 @@ local function get_formspec(tabview, name, tabdata)
 	end
 
 	local retval =
-		-- Search
+	-- Search
 		"field[0.25,0.25;7,0.75;te_search;;" .. core.formspec_escape(tabdata.search_for) .. "]" ..
 		"field_enter_after_edit[te_search;true]" ..
 		"container[7.25,0.25]" ..
 		"image_button[0,0;0.75,0.75;" .. core.formspec_escape(defaulttexturedir .. "search.png") .. ";btn_mp_search;]" ..
-		"image_button[0.75,0;0.75,0.75;" .. core.formspec_escape(defaulttexturedir .. "clear.png") .. ";btn_mp_clear;]" ..
-		"image_button[1.5,0;0.75,0.75;" .. core.formspec_escape(defaulttexturedir .. "refresh.png") .. ";btn_mp_refresh;]" ..
+		"image_button[0.75,0;0.75,0.75;" ..
+		core.formspec_escape(defaulttexturedir .. "clear.png") .. ";btn_mp_clear;]" ..
```

Comment:
Such changes could for example very well be considered problematic: This makes it less obvious what belongs to this image button element. It would be good if it was at least indented.